easybuild/easyblocks/c/cuda.py

```diff
@@ -136,8 +137,15 @@ def create_wrapper(wrapper_name, wrapper_comp):
         for comp in (self.cfg['host_compilers'] or []):
             create_wrapper('nvcc_%s' % comp, comp)
 
+        # ldconfig is usually in /sbin or /usr/sbin
+        curr_path = os.environ.get('PATH', '')
+        setvar('PATH', os.pathsep.join(['/sbin', '/usr/sbin', curr_path]))
```

Comment:
@akesandgren Maybe we should only do this if `ldconfig` was not available yet?

Also, we can actually make very sure whether `ldconfig` is in `/sbin` or `/usr/sbin` before prepending the location to `$PATH`, using `os.path.exists`.

Reply:
Well, that's what which does basically, so adding them and running which is shorter.

Reply:
I would prefer not pushing stuff into `$PATH` that we don't need, and I'd also prefer *appending* to `$PATH`, so something like this:

```python
ldconfig = which('ldconfig')
if ldconfig:
    self.log.info("ldconfig found at %s", ldconfig)
else:
    # ldconfig is usually in /sbin or /usr/sbin, so try and find it there
    for cand_path in ['/sbin', '/usr/sbin']:
        if os.path.exists(os.path.join(cand_path, 'ldconfig')):
            self.log.info("ldconfig found at %s, so appending that location to $PATH")
            setvar('PATH', os.pathsep.join(os.environ.get('PATH', ''), cand_path))
            break

    # fail if we couldn't find ldconfig, because it's really needed
    if not which('ldconfig'):
        raise EasyBuildError("Unable to find the 'ldconfig' in $PATH")
```

Maybe we should make a dedicated function for this in framework, it may come in useful in other places too (doesn't need to be a blocker for this PR, we can start using it here when it's in place)?